### ISSUE TITLE
Trying to give windows it's own IP

### ISSUE THREAD

**Issue by nirurin** [dockur/windows#637]
### Operating system

Unraid 6.12.8

### Description

Does anyone know how to set up an actual IP for the windows container? 

I have turned on the DHCP (Y) and set a custom network (both "custom br0" and a new custom, I tried both ways), but the custom br0 doesn't boot at all, and using my own custom network does boot into windows but there's no internet connection. 

Unless this just doesn't work with it's own IP? 

 br0 network throws this error = "RTNETLINK answers: Invalid argument
❯ ERROR: Cannot create macvtap interface. Please make sure that the network type is 'macvlan' and not 'ipvlan',
❯ ERROR: that your kernel is recent (>4) and supports it, and that the container has the NET_ADMIN capability set."

Edited 10 hours ago by Nirin

### Docker compose

The default docker file for the unraid CA version. I don't know how the compose is set up for it, but I believe it was made by yourself so hopefully you do!
All I changed was DHCP to "Y" and selecting different network types. Other than that I only set the correct username and password. 

### Docker log

root@Hex:~# docker logs WindowsinDocker
❯ Starting Windows for Docker v3.12...
❯ For support visit https://github.com/dockur/windows
❯ CPU: 13th Gen Intel Core TM i5 13600K | RAM: 20/32 GB | DISK: 1747 GB (fuseblk) | HOST: 6.1.74-Unraid...

❯ Warning: the filesystem of /storage is FUSE, this extra layer will negatively affect performance!
RTNETLINK answers: Invalid argument
❯ ERROR: Cannot create macvtap interface. Please make sure that the network type is 'macvlan' and not 'ipvlan',
❯ ERROR: that your kernel is recent (>4) and supports it, and that the container has the NET_ADMIN capability set.

**Comment by nirurin** [comment]
Hmm not sure if this fell through the cracks, or if it's just that nobody knows how to fix it lol.

**Comment by DasAoD** [comment]
Würde mich auch interessieren. :)
I would also be interested. :)

**Comment by nirurin** [comment]
Hmm adding another comment here @kroese as I'm not sure anyone has seen this...

**Comment by kracer86** [comment]
may be help
I had the same problem under Unraid.
Solution: 
Look under Docker Settings, Enable Advance View.
Under “Docker custom network type:” set to macvlan (instead of ipvlan)

**Comment by thestraycat** [comment]
Yeah, i'm not too keen on changing my whole unraid docker networking layer from IPVLAN to MACVLAN to support a single container. Anyone using a nicer workaround?
